mautic/mautic
# JSON object or JSON array returned, depending on whether the field is empty

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | Y
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
After we make a GET request to /api/fields/contact, it appears that an empty array is returned when there are no values in the associated field and if there are values in the field, Mautic returns a JSON object. The same case is an empty list of entities.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Make a GET request and check the response for Contacts
2. Properties attributes are arrays, if empty

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Make a GET request and check the response for Contacts
3. The response for both empty and non-empty properties attributes should be an object
